packages/adminrouter/extra/src/README.md

```diff
@@ -8,7 +8,10 @@ within the cluster.
 
 ## Routes
 
-Admin Router runs on both master and agent nodes, each with different configurations. From these NGINX config files, [Ngindox](https://github.com/karlkfi/ngindox) is used to generates swagger-like docs:
+Admin Router runs on both master and agent nodes, each with different
+configurations. From these NGINX config files,
```

Comment:
`s/different/different NGIX`. This makes it clear which files the next sentence refers to as I had to re-read.